src/Adafruit_SPIFlashBase.cpp

```diff
@@ -102,7 +102,7 @@ static const SPIFlash_Device_t possible_devices[] = {
     MB85RS64V, MB85RS1MT, MB85RS2MTA, MB85RS4MT,
 
     // Other common flash devices
-    AT25SF041, AT25DF081A};
+    AT25SF041, AT25DF081A, P25Q16H};
```

Comment:
please remove this from default list

Reply:
How come? If it's not in the default list, the default behaviour will be for flash access to fail and nobody will have any way of knowing how to fix it.

Done anyway.

Reply:
you can pass your own custom flash devices with begin()
https://github.com/adafruit/Adafruit_SPIFlash/blob/master/src/Adafruit_SPIFlashBase.h#L44